api/controllers/settings.js

```diff
@@ -0,0 +1,54 @@
+const auth = require('../auth'),
+      serverUtils = require('../server-utils'),
+      settingsService = require('../services/settings'),
+      objectPath = require('object-path');
+
+const doGet = req => auth.getUserCtx(req).then(() => settingsService.get());
+
+module.exports = {
+  get: (req, res) => {
+    doGet(req, res)
+      .then(settings => res.json(settings))
+      .catch(err => {
+        serverUtils.error(err, req, res, true);
+      });
+  },
+  // deprecated
```

Comment:
Any note on when we can fully remove it?

Reply:
I've added a couple of usages to the comment. It's used widely outside of webapp (medic-configurer, medic-reporter, etc) that are better off using the old api for now.